src/components/themed/SimpleTextInput.tsx

```diff
@@ -104,6 +110,7 @@ export const SimpleTextInput = React.forwardRef<SimpleTextInputRef, SimpleTextIn
     maxLength,
     returnKeyType,
     secureTextEntry,
+    selected,
```

Comment:
The logic below becomes simpler if we set a default to `false`.

Reply:
No, this was intentional  because we use the `null` state in various changes to hijack the "normal" handling in favor of the `active` prop.